crates/db_views_moderator/src/admin_purge_comment_view.rs

```diff
@@ -40,6 +40,11 @@ impl AdminPurgeCommentView {
       query = query.filter(admin_purge_comment::admin_person_id.eq(admin_person_id));
     };
 
+    // If a post or comment ID is given, then don't find any results
+    if params.post_id.is_some() || params.comment_id.is_some() {
```

Comment:
I *could've* externalized this, but since the two other cases (only checking for one of them) also exist, it doesn't seem worth it.

Reply:
Its good enough, I hope we can do a cleaner solution for this together with https://github.com/LemmyNet/lemmy/issues/2444